### PR TITLE
fsw: fix test timeout

### PR DESCRIPTION
**Pull request by ilovezfs**

### CODE REVIEW

**Comment by ilovezfs** [comment]
@xu-cheng This version of the test has now passed CI twice.

File: Formula/fsw.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Fsw < Formula
   homepage "https://emcrisostomo.github.io/fsw/"
   url "https://github.com/emcrisostomo/fsw/releases/download/1.3.9/fsw-1.3.9.tar.gz"
   sha256 "9222f76f99ef9841dc937a8f23b529f635ad70b0f004b9dd4afb35c1b0d8f0ff"
+  revision 1
```
Comment (xu-cheng) [inline Formula/fsw.rb]: You should remove this line.

Comment (ilovezfs) [inline Formula/fsw.rb]: Removed.

**Comment by xu-cheng** [comment]
👍

**Pull request closed without merging**